index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Trials Survey</title>
+    <link rel="stylesheet" href="/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+
+  <body>
+    <div id="container">
+      <div class="header">
+        <h1 id="title">Microverse Trials Survey</h1>
+        <p id="description"><em>This survey is to help improve the quality of the Microverse Trials for applicants</em></p>
+      </div>
+      <div class="form">
+        <form id="survey-form">
+          
+          <div>
+            <label id="name-label">Name</label>
+            <input id="name" type="text" class="formInput" placeholder="What is your name"/>
+          </div>
+  
+          <div>
+            <label id="email-label">Email</label>
+            <input id="email" type="text" class="formInput" placeholder="What is your Email Address"/>
+          </div>
+  
+          <div>
+            <label id="number-label">Number</label>
+            <input id="number" type="text" class="formInput" placeholder="Enter a Value" min="17" max="50"/>
+          </div>
+  
+          <div id="select">
+            <label>How was the collaboration during the trials?</label>
+            <select id="dropdown">
+              <option>Choose</option>
+              <option>Bad</option>
+              <option>Fair</option>
+              <option>Good</option>
+              <option>Excellent</option>
+            </select>
+          </div>
+  
+          <div id="radio">
+            <label>Did you learn form the collaboration?</label> 
```

Comment:
Kindly note that this `label` element has no attributes

Reply:
I think it would be better to add more explanation to your comments. This will help students to get why did you request changes on that point and it makes them inspired to get more information about the issue.